### PR TITLE
ID and Emag retexture + Uplink message bugfix

### PR DESCRIPTION
**Pull request by CrudeWax**
## About the PR 
Minor visual fix-ups for IDs, notably removing the non-transparent white space.
Updates emag to look more like the in-game ID cards
Updates emag's flavortext to mention the cooldown.
Fixes a visual bug when inserting telecrystals into the uplink popping up a un-decoded translation string.

**Screenshots**

image

**Changelog**

:cl: CrudeWax
- fix: fixed the popup for inserting TC into the uplink
- tweak: updated emag to look more like the rest of the ID cards

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Dupe https://github.com/space-wizards/space-station-14/pull/7084 ?

**Comment by CrudeWax** [comment]
Old one had issues with github. This one has more content and only the needed changes.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -125,7 +125,8 @@
   id: UplinkEmag
   category: Utility
   itemId: Emag
-  description: This sequencer, or "emag", can be used to open or hack a huge variety of items around the station.
+  listingName: Eletromagnetic Card
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:128]: why did the listing name have to be changed at all. its also misspelled

File: Resources/Locale/en-US/traitor/uplink/telecrystal-component.ftl [hunk 2]
```diff
@@ -1 +1 @@
-telecrystal-component-sucs-inserted = You slot {$source} into the {$target}.
\ No newline at end of file
+telecrystal-component-sucs-inserted = You slot the telecrystals into the uplink
```
Comment (mirrorcult) [inline Resources/Locale/en-US/traitor/uplink/telecrystal-component.ftl:1]: this is the incorrect way to fix this, pass in the args properly when this is called and then do something like this

```suggestion
telecrystal-component-sucs-inserted = You slot {THE($source)} into {THE($target)}.
```

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 3]
```diff
@@ -125,7 +125,8 @@
...
+  description: The business card of the syndicate, this sequencer is able to break open airlocks and tamper with a variety of station devices. Moderate cooldown between usages.
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:129]: i dont think this is even true, it just has a recharge so if you use them all up  you have to wait a bit

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
sprites look fine

**Comment by mirrorcult** [comment]
id prefer if the listing name was not changed

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**